include/aspect/material_model/drucker_prager.h

```diff
@@ -76,43 +76,17 @@ namespace aspect
      * @ingroup MaterialModels
      */
     template <int dim>
-    class DruckerPrager : public MaterialModel::InterfaceCompatibility<dim>, public ::aspect::SimulatorAccess<dim>
+    class DruckerPrager : public MaterialModel::Interface<dim>, public ::aspect::SimulatorAccess<dim>
```

Comment:
I don't see anything what should change based on this. Did you have anything in mind? I could add that the derivatives are added to the material model.